test/starlark_tests/apple_xcframework_import_tests.bzl

```
@@ -0,0 +1,67 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""apple_dynamic_xcframework_import and apple_static_xcframework_import Starlark tests."""
+
+load(
+    ":rules/analysis_target_outputs_test.bzl",
+    "analysis_target_outputs_test",
+)
+load(
+    ":rules/apple_verification_test.bzl",
+    "apple_verification_test",
+)
+
+def apple_xcframework_import_test_suite(name = "apple_xcframework_import"):
```

With a recent change, the name is no longer defaulted, and is instead passed in. The `Args` section also follows a standard format now.
```suggestion
def apple_xcframework_import_test_suite(name):
```